runestone/common/css/style.css

```diff
@@ -347,4 +347,13 @@ div.error {
 
 .correct {
     color: #84b8d9;
-}
\ No newline at end of file
+}
+
```

Comment:
Per ogni nuova modifica sarebbe il caso di indicare con un breve commento cos'è e cosa fa.
In questo caso metterei un commento generale per 
```
/* New rules for oep-libro-python <https://github.com/Open-Education-Polito/oep-python-interactive-book> */
```
e poi dei commenti per ogni nuova regola, giusto per capire cosa sono.
In questo caso è semplice ma è sempre meglio specificare.